data/bigquery_sharded_data_pull.py

```diff
@@ -0,0 +1,75 @@
+"""
+Pull data >1Gb from GCP Bigquery using sharding functionality.
+Based on:
+    https://cloud.google.com/bigquery/docs/exporting-data#exporting_table_data
+    https://cloud.google.com/storage/docs/reference/libraries#using_the_client_library
+
+Author: Travis Couture
+Created: 03/17/2020
+Last Updated: 03/17/2020
+"""
+
+import datatable as dt
+import pandas as pd
+import os
+from h2oaicore.data import CustomData
+from h2oaicore.systemutils import config
+from google.cloud import bigquery
+from google.cloud import storage
+from functools import reduce
+
+
+# Please fill before usage
+# Note that this information is logged in Driverless AI logs.
+
+BUCKET_NAME = ''
+PROJECT = ''
+DATASET_ID = ''
+TABLE_ID = ''
+WILDCARD_NAME = ''
```

Comment:
nit: I might maybe leave a note that this should be something like `/new_data_directory/data-*.csv` or just some example that works properly in dai context

Reply:
probably examples for all the variables would be nice.

Reply:
Added comments